code/core/src/main/java/com/adobe/marketing/mobile/internal/CoreConstants.kt

```diff
@@ -13,7 +13,7 @@ package com.adobe.marketing.mobile.internal
 
 internal object CoreConstants {
     const val LOG_TAG = "MobileCore"
-    const val VERSION = "2.2.2"
+    const val VERSION = "2.2.3"
```

Comment:
I am assuming these changes are due to the previous release was directly pushed to main.

Reply:
yes, this pr has the changes from the lightroom npe fix branch